shared-libs/validation/src/validation_utils.js

```diff
@@ -33,21 +47,49 @@ const parseStartDate = (duration) => {
   return moment().subtract(parsed).valueOf();
 };
 
-const exists = async (doc, fields, options = {}) => {
-  if (!fields.length) {
-    return Promise.reject('No arguments provided to "exists" validation function');
-  }
-  const requestOptions = fields.map(field => {
-    return { key: [`${field}:${lowerCaseString(doc[field])}`] };
+const getExistsRequestOptions = (fields, doc) => {
+  return fields.map(field => {
+    if (doc[field].includes(' ')) {
```

Comment:
As I mentioned [in the cht-datasource PR](https://github.com/medic/cht-core/pull/9625#discussion_r1939941973) we need to update our cht-datasource handling of keyed freetext queries that contain white-space.  

We cannot have this logic fall back to the `medic-client/reports_by_freetext` view query since that view will literally not exist when running on the server.